ui/src/views/LocalRules.vue

```diff
@@ -835,6 +835,10 @@ export default {
       services: [],
       localServices: [],
       roles: [],
+      serviceOptions: {
+        placeholder: this.$i18n.t("any"),
```

Comment:
Any it's like GREEN or RED, I don't think it needs a translation :thinking:

Reply:
I agree, remove the translation and add the placeholder.
We should add also a info tooltip to the describe the behavior